### DISCUSSION TITLE
Prodidows 1.22.4 has officially been updated!

### DISCUSSION THREAD

**Discussion by UltraThyme42** [UltraThyme42/prodidows#7] [Announcements]
New Updates: 1. Bugfixes 2. Restored the wheel and the bird/parrot. 3. Added a new outfit Known Issues: 1. The say something feature can't be added in the chat bar. 2. The Google Sign-In feature and the actual Multiplayer Mode can't work. 3. 2 Math toggling options can't exist due to the settings app being full of buttons and bars. 4. You can't change your skin tone, gender, or your name (not your nickname) inside the game. 5. More math sometimes doesn't boot up. 6. The world list's still empty.
![Screenshot 2024-06-01 191424](https://github.com/XPMUser/prodidows/assets/109283781/5eba7485-69ae-45ea-b487-01f8ee2fd174)

https://xpmuser.github.io/prodidows/1-22-4/?mods=WalkSpeed,FastGameSpeed

**Comment by Toonigy** [comment]
updated to bring back toontown battle and win themes
